debugger/websockets/packet_queue.c

```diff
@@ -52,25 +52,24 @@ bool packet_queue_empty (Emulator *emu)
   }
 }
 
-void packet_enqueue (Emulator *emu, void *item, size_t size, 
+void packet_enqueue (Emulator *emu, void *item, size_t size,
 		     uint8_t opcode)
-{  
-  // NOP
-  /*Server *s = emu->debugger->server;
+{
+  Server *s = emu->debugger->server;
   Packet *head, *tail, *cur;
   void *heap_item;
-  
+
   while (s->spin_lock);
   s->spin_lock = true;
 
   if (s->pending_packets_head == NULL) {
-    head = tail = s->pending_packets_head = s->pending_packets_tail = 
-      (Packet *) calloc(1, sizeof(Packet));        
+    head = tail = s->pending_packets_head = s->pending_packets_tail =
+      (Packet *) calloc(1, sizeof(Packet));
```

Comment:
Missing `free`

Reply:
queue item is freed in packet_dequeue;
queue item message WAS at some point in time freed, but this is commented out (no idea why, but probably didn't work)

We don't use this too much and we don't have tests for it... I prefer to leave it as it

Reply:
ok